src/tls.rs

```diff
@@ -25,6 +25,18 @@
 // NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
 // SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
+#[cfg(windows)]
+extern crate winapi;
```

Comment:
Do you really need this?

Reply:
As in, you generally don't need to explicitly import crates that are defined in `Cargo.toml`, though I don't know if this is special in some way.